pil-analyzer/src/untyped_evaluator.rs

```diff
@@ -19,6 +19,7 @@ pub fn evaluate_expression_to_int(
     evaluator::evaluate_expression::<GoldilocksField>(
         &ExpressionProcessor::new(driver, &Default::default()).process_expression(expr),
         driver.definitions(),
+        driver.solved_impls(),
```

Comment:
At this point, we have not even run the type checker, so the solved_impls will always me empty.

I think you can remove the function and just pass Default::default() here.